src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/Projections/LiveVacancy/WageType.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.LiveVacancy
+{
+    public enum WageType

```

Comment:
Probably one to debate between us but should we just use the Domain enums?

Reply:
Yeah, Sachin raised this yesterday too.  I've spoken to Mark this morning. As the projection is a read-only view of the vacancy then these values are not going to change so having an enum of possible options doesn't really make sense in this context.  So I'm going to just turn the enums into strings instead for the projection.